<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmployeeWindow</class>
 <widget class="QWidget" name="EmployeeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1600</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Employee Management System</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    EmployeeWindow {
        background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
            stop:0 #f8f9fa, stop:1 #e9ecef);
    }
    QGroupBox {
        border: 2px solid #dee2e6;
        border-radius: 8px;
        margin-top: 20px;
        padding-top: 15px;
        font: bold 14px 'Segoe UI';
        color: #2b2d42;
        background: white;
    }
    QPushButton {
        background-color: #6c757d;
        color: white;
        border: none;
        padding: 10px 20px;
        border-radius: 5px;
        font: 13px 'Segoe UI';
        min-width: 120px;
    }
    QPushButton:hover {
        background-color: #5a6268;
    }
    QPushButton:pressed {
        background-color: #545b62;
    }
    QTableView {
        background: white;
        border: 2px solid #dee2e6;
        selection-background-color: #cce3ff;
        alternate-background-color:#dadde0;
        color: #2b2d42;
    }
    QTableView::item:selected {
        background-color: #5d6294;
    }
    QTableView::item:!selected:alternate {
        background-color: #dadde0;
    }
    QHeaderView::section {
        background: #6c757d;
        color: white;
        padding: 8px;
        border: none;
    }
    QLineEdit, QComboBox {
        background-color: black;
        border: 2px solid #ced4da;
        border-radius: 4px;
        padding: 6px;
        font: 13px 'Segoe UI';
    }
    QLabel {
        font: 13px 'Segoe UI';
        color: #495057;
    }
    QChartView {
        background: white;
        border: 1px solid #dee2e6;
        border-radius: 5px;
        margin: 5px;
    }
   </string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>1560</width>
     <height>860</height>
    </rect>
   </property>
   <widget class="QWidget" name="listPage">
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>1100</width>
       <height>500</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCreate">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>590</y>
       <width>160</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>‚ûï New Employee</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnUpdate">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>590</y>
       <width>160</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>‚úèÔ∏è Edit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDelete">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>590</y>
       <width>160</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>‚ùå Delete</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPDF">
     <property name="geometry">
      <rect>
       <x>1130</x>
       <y>130</y>
       <width>160</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>üìä Generate PDF</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnStats">
     <property name="geometry">
      <rect>
       <x>1130</x>
       <y>210</y>
       <width>160</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>üìà Statistics</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtSearch">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>300</width>
       <height>35</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Search employees...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnBack">
     <property name="geometry">
      <rect>
       <x>940</x>
       <y>30</y>
       <width>160</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>üè† home</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="formPage">
    <widget class="QGroupBox" name="grpEmployeeForm">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>100</y>
       <width>1200</width>
       <height>600</height>
      </rect>
     </property>
     <property name="title">
      <string>Employee Details</string>
     </property>
     <widget class="QLineEdit" name="txtUsername">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>300</width>
        <height>35</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtPassword">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>300</width>
        <height>35</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Password</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtFirstName">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>230</y>
        <width>300</width>
        <height>35</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>First Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtLastName">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>290</y>
        <width>300</width>
        <height>35</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Last Name</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cmbRole">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>350</y>
        <width>300</width>
        <height>35</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Select Role</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtPhone">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>410</y>
        <width>300</width>
        <height>35</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Phone Number</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSubmitForm">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>470</y>
        <width>160</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>üíæ Save</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCancelForm">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>470</y>
        <width>160</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>‚ùå Cancel</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="statsPage">
    <widget class="QGraphicsView" name="chartView">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>800</width>
       <height>500</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnBackStats">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>610</y>
       <width>160</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
                      background-color: #6c757d;
                      color: white;
                      border-radius: 5px;
                  </string>
     </property>
     <property name="text">
      <string>‚Üê Back to List</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
